### PR TITLE
Initial commit for final report

### PR DESCRIPTION
**Pull request by archbung**
Add

- Summary of third paper
- Introduction
- Context and Connection

### CODE REVIEW

**Comment by nzyuzin** [comment]
There's a file for the report already, please keep your text in it so we don't have to do anything extra later.
https://github.com/nzyuzin/static-analysis-papers/blob/master/report/report.tex

It's now a copy-paste from the summary and you're expected to overwrite your summary with the report.

**Review by nzyuzin — CHANGES_REQUESTED** [review]
There's a file for the report already, please keep your text in it so we don't have to do anything extra later.
https://github.com/nzyuzin/static-analysis-papers/blob/master/report/report.tex

It's now a copy-paste from the summary and you're expected to overwrite your summary with the report.

**Comment by archbung** [comment]
Renamed `final_report.tex` to `report.tex`

**Comment by nzyuzin** [comment]
Now you removed the material that doesn't relate to your section.

I have many changes there not yet committed and it would then cause awful conflicts.

Can you _only_ change the sections that relate to your paper leaving everything else as-is?

Upd: of course the introduction can be placed before everything, but removing other material is still not ok.

**Comment by archbung** [comment]
How about the Introduction section and the Context and Connection section?  I have added something to it also.

**Comment by nzyuzin** [comment]
Sure, it's ok.

The point is to leave unchanged the stuff on which Joachim or I may still be working.

**Comment by archbung** [comment]
Ok, I copied the summaries of the first two papers from `summary.tex` over to `report.tex`

File: report/report.tex [hunk 1]
```diff
@@ -583,314 +615,155 @@ \subsection{Liquid Type Inference}
 
 \subsection{Experiments}
 
+\begin{figure}
+    %\includegraphics[width=0.9\textwidth]{experiments.png}
+    \caption{Experimental Results}
+    \label{fig:exp}
+\end{figure}
```
Comment (nzyuzin) [inline report/report.tex]: Did you intend to make these changes?

File: report/report.tex [hunk 2]
```diff
@@ -583,314 +615,155 @@ \subsection{Liquid Type Inference}
...
+
 In order to assess the advantage of this approach liquid type inference is implemented in \textbf{DSolve}.
 It is tested over a set of benchmarks from the \textbf{DML} project~\cite{Xi:1998:EAB}.
 To allow array bounds checking the qualifiers are of the form $\nu \bowtie X$ where $\bowtie \in \{<, \le, =, \neq, \geq, >\}$ and $X \in \{0, \star,$ \lstinline{len} $\star$\}.
-In most cases \textbf{DSolve} can prove the safety of array accesses without additional annotations i.e. qualifiers.
+The results are summarized in Figure~\ref{fig:exp}.
+The columns for \textbf{DML} and \textbf{DSolve} state how many lines respectively characters of manual annotations are need.
+We observe that in most cases \textbf{DSolve} can prove the safety of array accesses without additional annotations i.e. qualifiers.
```
Comment (nzyuzin) [inline report/report.tex]: Here too.

**Review by nzyuzin — APPROVED** [review]

**Pull request merged**